src/components/modules/AccountPickers/cmp.tsx

```diff
@@ -0,0 +1,206 @@
+            variant={button.variant(true)}
+          >
+            <Button
+              disabled={oneNetwork}
+              as="button"
+              kind={button.kind(true)}
+              color={button.color(true)}
+              variant={button.variant(true)}
+              size="md"
+            >
+              <Icon name={selectedNetwork.icon} size="1.5em" prefix="custom" />
+            </Button>
+            {!oneNetwork && <Icon name="angle-down" size="1em" />}
+          </Button>
+          <Button
+            ref={walletPickerTriggerRef}
+            as="button"
+            kind={button5.kind}
+            color={button5.color}
+            variant={button5.variant}
+            size="md"
+            onClick={handleDisplayWalletPicker}
+          >
+            <div tw="flex items-center gap-3">
+              {accountAddress
+                ? ensName || ellipseText(accountAddress, 6, 4)
+                : 'Connect'}
+            </div>
+          </Button>
+        </div>
+      ) : (
+        <>
+          <Button
+            ref={condensedPickerTriggerRef}
+            as="button"
+            kind={button5.kind}
+            color={button5.color}
+            variant={button5.variant}
+            size="md"
+            onClick={handleDisplayCondensedPicker}
+          >
+            <div tw="flex items-center gap-3">
+              <StyledIcon
+                $network={selectedNetwork}
+                $isConnected={isConnected}
+                $isMobile={isMobile}
+              />
+            </div>
+          </Button>
+        </>
+      )}
+
+      {displayWalletPicker &&
+        createPortal(
+          <StyledPicker
+            $isOpen={walletPickerOpen}
+            $position={walletPosition}
+            ref={walletPickerRef}
+          >
+            {isConnected && (
+              <>
+                <AccountInformation
+                  rewards={rewards}
+                  accountAddress={accountAddress}
+                  accountAddressHref={accountAddressHref}
+                  balance={accountBalance}
+                />
+                <StyledLine />
+              </>
+            )}
+            <WalletSelector
+              isConnected={isConnected}
+              onConnect={handleConnect}
+              onDisconnect={handleDisconnect}
+              selectedNetwork={selectedNetwork}
+            />
+          </StyledPicker>,
+          document.body,
+        )}
+      {displayNetworksPicker &&
+        createPortal(
+          <StyledPicker
+            ref={networksPickerRef}
+            $isOpen={networksPickerOpen}
+            $position={networksPosition}
+          >
+            <NetworkSelector
+              networks={networks}
+              selectedNetwork={selectedNetwork}
+              onSelectNetwork={handleSwitchNetwork}
+            />
+          </StyledPicker>,
+          document.body,
+        )}
+      {displayCondensedPicker &&
+        createPortal(
```

Comment:
Use our `<Portal>` component from `modules/Portal/` instead to make sure this component is 100% compatible with nextjs SSR

Reply:
Sure! Didn't know about that!
Thanks for spotting 🙌🏻